scarlet2/initialization.py

```diff
@@ -0,0 +1,331 @@
+# ---------------------------------------------------------------------------- #
+# Initialization routine for Scarlet2. We define function allowing for the     #
+# initialization of source morphologies and spectrums of indentified sources.  #
+#                                                                              #
+# For morphologies, we input the observational image with the locations of the #
+# sources calculated. Then we fit a 2D Gaussian to the cutout and optimize for #
+# the best fit with a maximum amplitute of 1.                                  #
+#                                                                              #
+# For spectrums, we input the observational image as with the morphology, we   #
+# just take the pixel value of the center of the source for each band.         #
+# ---------------------------------------------------------------------------- #
+
+import numpy as np
+import numpy.ma as ma
+import jax.numpy as jnp
+import jax
+from .plot import cut_square_box
+from .morphology import GaussianMorphology
+from jaxopt import ScipyBoundedMinimize
+import dm_pix
```

Comment:
Another dependency I like to avoid. You using it to rotate a Gaussian, which can be done analytically. #45 will bring this functionality.

Reply:
Removed fitting routine